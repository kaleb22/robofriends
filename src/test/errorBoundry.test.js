import { shallow} from 'enzyme'
import React from 'react'
import ErrorBoundry from '../components/ErrorBoundry'

it('renders ErrorBoundry component', () => {
  expect(shallow(<ErrorBoundry />)).toMatchSnapshot();
})

it('has an error', () => {
  
  const wrapper = shallow(<ErrorBoundry/>);
  expect(wrapper.instance().componentDidCatch()).toEqual(undefined);
})