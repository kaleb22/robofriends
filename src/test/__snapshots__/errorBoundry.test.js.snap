// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has an error 1`] = `ShallowWrapper {}`;

exports[`renders ErrorBoundry component 1`] = `ShallowWrapper {}`;
